### PR TITLE
Tests for updating multiple nested embeddeds

### PR DESCRIPTION
**Pull request by f-wrobel**

### CODE REVIEW

File: test/github-issues/1200/entity/A.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import {Entity} from "../../../../src/decorator/entity/Entity";
+import {Column} from "../../../../src/decorator/columns/Column";
+import {PrimaryGeneratedColumn} from "../../../../src/decorator/columns/PrimaryGeneratedColumn";
+import {B} from "./B";
+import {C} from "./C";
+
+@Entity()
+export class A {
```
Comment (pleerock) [inline test/github-issues/1200/entity/A.ts]: sorry for being annoying but "A", "B", "C", "n" arent a good names, its hard to work with such meaningless names. Plus they will appear in the top of the suggestions lists in IDEs when you search something with "A" or "B" or "C" first-letter name.

File: test/github-issues/1200/issue-1200.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import {Connection} from "../../../src/connection/Connection";
+import {A} from "./entity/A";
+import {expect} from "chai";
+
+describe("github issues > #1200: Pull request: Tests for updating multiple nested embeddeds", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        enabledDrivers: ["postgres"]
```
Comment (pleerock) [inline test/github-issues/1200/issue-1200.ts]: why only postgres is enabled?

**Comment by pleerock** [comment]
Also, why do you have this test failing?:

```
  AssertionError: expected { Object (b, c, ...) } to deeply equal { Object (id, n, ...) }
      + expected - actual
       {
         "b": {
           "c": {
             "d": {
      -        "n": 4
      +        "n": 14
             }
      -      "n": 3
      +      "n": 13
           }
           "n": 12
         }
         "c": {
           "d": {
      -      "n": 6
      +      "n": 16
           }
           "n": 15
         }
         "id": 1
      
```

**Comment by f-wrobel** [comment]
Sorry, I messed this up. Please, hold on with merging, I'm going to fix that.

**Comment by pleerock** [comment]
Sounds good

**Comment by pleerock** [comment]
@f-wrobel any updates?

**Comment by pleerock** [comment]
Closing because of no answer and activity from developer.

**Comment by f-wrobel** [comment]
Hi, it's been a long time since then :) Surprisingly, I've just created a new PR for this issue, and I hope it can be merged without any concern this time.

**Pull request closed without merging**